app/src/pages/inside/testPlansPage/milestones/milestoneModals/duplicateMilestoneModal/duplicateMilestoneModal.scss

```
@@ -0,0 +1,22 @@
+@use '../../milestonesTable/milestonesShared' as ms;
+
+.duplicate-milestone-modal {
+  &__notice {
+    margin-bottom: 16px;
+
+    :global([class*='system-message'])
```

Stylelint (17.5.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)